x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/index.ts

```diff
@@ -23,4 +29,14 @@ export const registerTrustedAppsRoutes = (
     },
     getTrustedAppsListRouteHandler(endpointAppContext)
   );
+
+  // CREATE
+  router.post(
+    {
+      path: TRUSTED_APPS_CREATE_API,
+      validate: PostTrustedAppCreateRequestSchema,
```

Comment:
if the schema doesn't validate here, we get a 400 error?  Do we get a meaningful message from the error?  From your tests, it looks like an error will be thrown as expected - `expect(() => body.validate(bodyMsg)).toThrow()` - just wondering what the error actually looks like

Reply:
they are "typical" errors thrown by the schema lib. Here are a few examples:

```json5
// entries was an empty array
{
  'statusCode': 400,
  'error': 'Bad Request',
  'message': '[request body.entries]: array size is [0], but cannot be smaller than [1]',
}
```

```json5
// enty had os set to `winz`
{
  'statusCode': 400,
  'error': 'Bad Request',
  'message': '[request body.os]: types that failed validation:\n- [request body.os.0]: expected value to equal [linux]\n- [request body.os.1]: expected value to equal [macos]\n- [request body.os.2]: expected value to equal [windows]',
}
```